keyboards/handwired/eagleii/readme.md

```diff
@@ -0,0 +1,15 @@
+# Eagle II Keyboard
+
+![Eagle II](https://upload.wikimedia.org/wikipedia/en/a/ae/Eagleiicomp.jpg)
+
+A short description of the keyboard/project
+
+* Keyboard Maintainer: [Spaceman](https://github.com/rionlion100)
+* Hardware Supported: Eagle II Keyboard
+* Hardware Availability: Not availible 
+
+Make example for this keyboard (after setting up your build environment):
+
+    make eagleii:default
```

Comment:
The `make` command is case-sensitive, and has to match the path in relation to `qmk_firmware/keyboards/`.

```suggestion
    make handwired/eagleii:default
```